src/libNode/PoWProcessing.cpp

```diff
@@ -287,33 +287,13 @@ bool Node::SendPoWResultToDSComm(const uint64_t& block_num,
       break;
     }
   }
-  //[FIX-ME] Send to PoW PACKET_SENDERS + 1
+  // Send to PoW PACKET_SENDERS + DS Leader
```

Comment:
FIX ME - was added here to fix the number of total nodes to send including leader. right now its NUM_NODES_TO_SEND_LOOKUP + Leader which needs t be fixed eventually.